src/themes/dark.scss

```diff
@@ -1,3 +1,11 @@
 .wxc-theme-dark {
   --wxc-meeting-background: linear-gradient(180deg, #1A1A1A 0%, #0F0F0F 100%);
+  // Icons
+  --wxc-audio-microphone-icon-interior-fill-color: #27A17A;
+  --wxc-audio-microphone-icon-fill-color: #fff;
+  --wxc-camera-icon-interior-fill-color: #27A17A;
+  --wxc-camera-icon-fill-color: #fff;
+  --wxc-settings-icon: #fff;
+  --wxc-share-screen-icon-fill-color: #fff;
+  --wxc-cancel-icon-fill-color: #fff;
```

Comment:
let's prefix everything with `--wxc-icon--`  like in the comment above. The "fill" word is not really adding much information.